tenancy-api-mapping/openapispecs/generated/amc-app-orch-deployment-app-deployment-manager-openapi.yaml

```diff
@@ -1,7 +1,3 @@
-# SPDX-FileCopyrightText: 2025 Intel Corporation
-#
-# SPDX-License-Identifier: Apache-2.0
```

Comment:
We should add a `.license` file instead so the copyright is not removed everytime the file gegenerated: https://reuse.software/faq/#uncommentable-file

Reply:
Done.